internal/driver/glfw/window_test.go

```diff
@@ -39,7 +39,9 @@ func TestMain(m *testing.M) {
 
 func TestGLDriver_CreateWindow(t *testing.T) {
 	w := NewGLDriver().CreateWindow("Test").(*window)
+	wt := NewGLDriver().CreateWindow("").(*window)
```

Comment:
This line effectively sets up a new test case unrelated to the test in here already. Could you move it to a new test, perhaps named TestGLFriver_CreateWindow_EmptyTitle? Then the window bar could return to the common “w”